server/controllers/finance/reports/cash/receipt.handlebars

```diff
@@ -1,41 +1,50 @@
 {{> head title="CASH.TITLE" }}
 
-<div class="container" style="font-size: 0.8em;">
+<div class="container" style="font-size: 0.9em;">
```

Comment:
In the future we should clearly define font family, size and style uniformly across all documents.